apps/src/codebridge/Workspace/WorkspaceAndConsole.tsx

```diff
@@ -80,7 +81,7 @@ const WorkspaceAndConsole: React.FunctionComponent = () => {
         resizeItemTop={() => PANEL_TOP_COORDINATE}
         position={editorHeight}
         onResize={handleResize}
-        style={{position: 'static'}}
+        style={{position: 'static', backgroundColor: color.light_gray_950}}
```

Comment:
should this be pulled out into its own class?

Reply:
`HeightResizer` is an old component that doesn't have the option to pass in a class name, and it uses styles that we are partially overriding here (position). I'm going to leave as-is for now, that component could use a refactor away from radium 😄